plugins/brew/README.md

```diff
@@ -16,4 +16,6 @@ plugins=(... brew)
 | brewsp | `brew list --pinned` | Show the versions of pinned formulae, or only the specified (pinned) formulae if formulae are given. |
 | bubo   | `brew update && brew outdated` | Fetch the newest version of Homebrew and all formulae, then list outdated formulae. |
 | bubc   | `brew upgrade && brew cleanup` | Upgrade outdated, unpinned brews (with existing install options), then removes stale lock files and outdated downloads for formulae and casks, and removes old versions of installed formulae. |
+| bcbu   | `brew cleanup && brew upgrade` | Reverse-order of the above so that the least outdated elements of the above list are kept, after an upgrade of outdated, unpinned brews. |
```

Comment:
Maybe [`-s`](https://docs.brew.sh/Manpage#commands) would make sense in that scenario as well: purging everything but the _least_ out-dated stuff?